ttnn/cpp/ttnn/operations/embedding/device/kernels/compute/tilize.cpp

```diff
@@ -0,0 +1,27 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+
+#include "compute_kernel_api/tilize.h"
```

Comment:
Can we put this kernel underneath tilize op? nothin in this kernel seems embedding specific.

Reply:
I can move it to tilize op. I originally put this here because the original tilize kernel had hardcoded CB ids, so I created this new one to take them in as arguments while leaving the old one to not affect other ops.